cli/Makefile

```diff
@@ -1,21 +1,25 @@
 SHELL := /bin/bash
 
 # Changing this value will trigger a new release
-VERSION=v0.6.2
+VERSION=v0.7.0
 BINARY=bin/cft
 GITHUB_REPO=github.com/GoogleCloudPlatform/cloud-foundation-toolkit
 PLATFORMS := linux windows darwin
 BUILD_DIR=./bin
+SCHEMA_DIR=./bpmetadata/schema
 NAME=cft
 BUCKET=gs://cft-cli
 
 # Setup the -ldflags option for go build here, interpolate the variable values
 LDFLAGS=-ldflags "-X $(GITHUB_REPO)/cli/cmd.Version=$(VERSION)"
 
+# need an extra build command to embed an most up to date schema in the binary
 .PHONY: build
 build:
 	go build ${LDFLAGS} -o ${BUILD_DIR}/${NAME}
-
+	cd bpmetadata/schema && go build ./...
+	./bpmetadata/schema/schema -output=bpmetadata/schema
```

Comment:
Shouldn't this be before building the CFT binary at L19?

Reply:
We need the metadata built before the schema binary can generate the up to date schema.